src/napari_spatialdata/_viewer.py

```diff
@@ -434,7 +434,8 @@ def add_sdata_image(self, sdata: SpatialData, key: str, selected_cs: str, multi:
         affine = _get_transform(sdata.images[original_name], selected_cs)
         rgb_image, rgb = _adjust_channels_order(element=sdata.images[original_name])
 
-        channels = get_channels(sdata.images[original_name])
+        channels = (0,) if rgb else get_channels(sdata.images[original_name])
```

Comment:
```suggestion
        channels = ("RGB(A),) if rgb else get_channels(sdata.images[original_name])
```
With #153 we have different behaviour now for detecting whether something is supposed to be RGB(A). When parsing Image2D/3DModel users can specify c_coords as ('r','g','b') with optionally alpha to indicate it is a RGB image. I would prefer to just have the channel specified as `RGB(A)` in that case similar to colormap of the napari image layers when it is specified to be a RGB image.